tensorflow/tensorflow_plugin.cc

```diff
@@ -992,7 +996,7 @@ struct TensorflowGraphBase: public Function {
             return castToTypedTensor(val, storageToDatatype(storageType));
         }
 
-        cerr << "trying to cast " << jsonEncode(val) << " to tensor" << endl;
+        //cerr << "trying to cast " << jsonEncode(val) << " to tensor" << endl;
         throw HttpReturnException(500, "Unable to cast value to tensor");
```

Comment:
Let's add a little more information (some idea of what kind of expression) to help in debugging down the road.